content/tutorial/tutorial.md

```diff
@@ -213,60 +213,60 @@ class Square extends React.Component {
 }
 ```
 
-Before:
+Przed zmianą:
 
 ![React Devtools](../images/tutorial/tictac-empty.png)
 
-After: You should see a number in each square in the rendered output.
+Po zmianie: W każdym wyrenderowanym polu powinien być wyświetlany kolejny numer.
 
 ![React Devtools](../images/tutorial/tictac-numbers.png)
 
-**[View the full code at this point](https://codepen.io/gaearon/pen/aWWQOG?editors=0010)**
+**[Zobacz dotychczasowy kod](https://codepen.io/gaearon/pen/aWWQOG?editors=0010)**
 
-Congratulations! You've just "passed a prop" from a parent Board component to a child Square component. Passing props is how information flows in React apps, from parents to children.
+Gratulacje! Właśnie udało Ci się "przekazać atrybut" z komponentu nadrzędnego `Board` do komponentu potomnego `Square`. Przekazywanie atrybutów to jeden ze sposobów na przepływ danych między rodzicem a dzieckiem w drzewie.
 
-### Making an Interactive Component {#making-an-interactive-component}
+### Tworzenie interaktywnego komponentu {#making-an-interactive-component}
 
-Let's fill the Square component with an "X" when we click it. 
-First, change the button tag that is returned from the Square component's `render()` function to this:
+Niech teraz komponent `Square` wypełnia się literą "X" po kliknięciu w niego.
+Najpierw dokonaj zmian w elemencie `<button />` zwracanym przez `Square` w metodzie `render()`:
 
 ```javascript{4}
 class Square extends React.Component {
   render() {
     return (
-      <button className="square" onClick={function() { alert('click'); }}>
+      <button className="square" onClick={function() { alert('klik'); }}>
```

Comment:
```suggestion
      <button className="square" onClick={function() { alert('kliknięcie'); }}>
```

Reply:
W "alercie" raczej umieściłbym "klik" niż "kliknięcie". Ewentualnie "kliknięto w przycisk".

Reply:
"kliknięto w przycisk" 👍